draft-packages/menu/KaizenDraft/Menu/StatelessMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import { default as React, ReactElement } from "react"
+
+import { GenericMenuProps, render } from "./Menu"
+import MenuDropdown from "./MenuDropdown"
+const styles = require("./styles.scss")
+
+export type StatelessMenuProps = {
+  isMenuVisible: boolean
+  toggleMenuDropdown: () => void
+  hideMenuDropdown: () => void
+  renderButton: (args: {
+    onClick: () => void
+    onMouseDown: (e: any) => void
```

Comment:
I'm not clear on why `onMouseDown` takes an event argument here but `onClick` doesn't.

Reply:
Ah, now that you mention it, I think it's actually a potential bug. `e.stopPropagation()` was being called on `onClick` in the original version, but I didn't notice it as it was part of the `toggleDropdown` function. It doesn't seem to make any difference in practice, but I will update it - there must have been a reason for it. Good catch!